### ISSUE TITLE
Typeahead use react-window/virtualized

### ISSUE THREAD

**Issue by GarvinH** [TBS-EACPD/infobase#881]
related #815

**Comment by GarvinH** [comment]
Success: virtualizing list items

Failure: keyboard nav

- Since the list elements don't render, navigating straight to the bottom (i.e. pressing the up arrow to skip to the bottom) does not work

To "solve" this problem, we can manually change the `scrollToRow` value on `<List />` or we can use `ArrowKeyMapper`

with manually changing, navigating straight to the bottom does not work

with `ArrowKeyMapper`, navigating to the bottom works HOWEVER

- The active row cannot be obtained from the components of `react-virtualized`
- Focus on the list is required (instead of focus on the input element)

**Comment by GarvinH** [comment]
I may or may not have found a work around solution.

Gonna reopen this temporarily and will post my findings again if it fails to work
